### ISSUE TITLE
[BUG]: Rocket Fuel/Oxidizer Loader automation output doesn't do anything when loading liquid fuels/oxidizers

### ISSUE THREAD

**Issue by isochronous** [Sgt-Imalas/Sgt_Imalas-Oni-Mods#131]
### Which Mod?

Rocketry Expanded

### Description of the bug/crash

The automation output port on the rocket fuel and oxidizer loaders does not actually output a green signal when liquid fuel is being loaded onto the rocket, contrary to the automation port's tooltip.

### Steps To Reproduce

Load some liquid hydrogen into a rocket via the rocket fuel loader. Note that the output never goes green.
Same process for liquid oxygen.

### Player.log

[Player.log](https://github.com/user-attachments/files/17081868/Player.log)

**Comment by Sgt-Imalas** [comment]
![image](https://github.com/user-attachments/assets/56fc50cf-5eb4-4640-b255-f1814c6e17c9)
cannot replicate, works as intended.
if you have multiple rocket platforms attached to a loader, its output signal will flicker (vanilla issue), this can be circumvented by using a buffer gate

**Comment by isochronous** [comment]
Yeah, I've got like six platforms hooked up. The signal doesn't even flicker for me, it just stays red :/

**Comment by isochronous** [comment]
My loader "network" is pretty big compared to vanilla setups. I wouldn't think the size would really have much effect on things except when adding/removing bits.  Also running with Fast Track, if that could possibly make a difference.
![image](https://github.com/user-attachments/assets/c76c86e3-aa64-4649-a271-df2fbab66d68)

**Comment by isochronous** [comment]
Began working properly after a restart :shrug:

**Comment by isochronous** [comment]
Okay, this just seems really flaky for me

https://github.com/user-attachments/assets/b8ee305a-22fb-41b0-9144-743cf4f047c3

**Comment by Sgt-Imalas** [comment]
the bug is potentially fixed
